src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -2273,22 +2269,32 @@ private SyntaxTrivia ScanWhitespace()
 
                 if (width < MaxCachedTokenSize)
                 {
-                    return _cache.LookupTrivia(
+                    var trivia = _cache.LookupTrivia(
                         TextWindow.CharacterWindow,
                         TextWindow.LexemeRelativeStart,
                         width,
```

Comment:
nit: could consider passing in a ROM? (feel free to consider that out of scope).

Reply:
I'm going to pass on this suggestion unless someone from the compiler team pipes in with approving that modification